### PR TITLE
(german translation) add missing liquid-container

### PR DESCRIPTION
**Pull request by kuhnchris**
For some reason the liquid container has been missing in the translation file (at all)

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR function correctly in-game, if applicable.

---
Should we compare the translation files with the core "bundle" file to see if languages are still missing various translation strings? Should be doable rather quickly with a little python (or even bash) script. Let me know - thanks!

### CODE REVIEW

**Comment by Catchears** [comment]
Anuke has such a script for all bundles, but the last time it was run is [over a year ago](https://github.com/Anuken/Mindustry/commit/f81f4ded03d2fd1ea9c275265879012efd031def#diff61ce9d1b28cacbd776d27aec77746028287057c61c4f9b70c28b9ab4da87bc48) now (or the last time it had an effect on this bundle).

**Comment by Catchears** [comment]
I would suggest moving the description above the liquid tank description to stay close to the English bundle. I'm working on a PR to get everything back up to date right now, but this should be merged first once the order has been corrected.

File: core/assets/bundles/bundle_de.properties [hunk 1]
```diff
@@ -1510,9 +1510,9 @@ block.conduit.description = Transportiert Flüssigkeiten. Wird mit Extraktoren,
 block.pulse-conduit.description = Transportiert Flüssigkeiten. Transportiert Flüssigkeiten schneller und speichert mehr als ein Leitungsrohr.
 block.plated-conduit.description = Transportiert Flüssigkeiten. Nimmt keine Flüssigkeiten von der Seite an.\nHat keine Lecks.
 block.liquid-router.description = Verteilt Flüssigkeiten auf bis zu drei Richtungen. Speicher außerdem eine kleine Menge an Flüssigkeit.
-block.liquid-container.description = Stores a sizeable amount of liquid. Outputs to all sides, similarly to a liquid router.
-block.liquid-tank.description = Speichert eine große Menge an Flüssigkeiten. Ähnlich wie ein Flüssigkeitsverteiler.
-block.liquid-junction.description = Fungiert als Brücke über zwei kreuzende Kanäle.
+block.liquid-container.description = Speichert etwas Flüssigkeit. Hat Ausgänge an allen Seiten, ähnlich einem Flüssigkeitsverteiler.
+bblock.liquid-tank.description = Speichert eine große Menge an Flüssigkeiten. Ähnlich wie ein Flüssigkeitsverteiler.
+lock.liquid-junction.description = Fungiert als Brücke über zwei kreuzende Kanäle.
```
Comment (Catchears) [inline core/assets/bundles/bundle_de.properties]: Vorsicht, hier hast du dich glaube ich vertippt.

**Comment by kuhnchris** [comment]
Sorry, you are correct - thanks! Reordered it.
Want me to squish the commit or is it fine like that?

Thanks,
Chris

File: core/assets/bundles/bundle_de.properties (hunk 1 above)
Comment (kuhnchris) [reply core/assets/bundles/bundle_de.properties]: einem=>einen? War mir selber nicht sicher. Am besten ich nehm den Text drunter, hast recht...

**Comment by Catchears** [comment]
Looks all good. Anuke will normally squish it himself if he wants it squished, not sure if he prefers us to do it though.

**Comment by kuhnchris** [comment]
Thanks, I'll leave it at that, if you guys need anything more let me know ;-)

File: core/assets/bundles/bundle_de.properties [hunk 2]
```diff
@@ -1510,7 +1510,7 @@ block.conduit.description = Transportiert Flüssigkeiten. Wird mit Extraktoren,
 block.pulse-conduit.description = Transportiert Flüssigkeiten. Transportiert Flüssigkeiten schneller und speichert mehr als ein Leitungsrohr.
 block.plated-conduit.description = Transportiert Flüssigkeiten. Nimmt keine Flüssigkeiten von der Seite an.\nHat keine Lecks.
 block.liquid-router.description = Verteilt Flüssigkeiten auf bis zu drei Richtungen. Speicher außerdem eine kleine Menge an Flüssigkeit.
-block.liquid-container.description = Stores a sizeable amount of liquid. Outputs to all sides, similarly to a liquid router.
+block.liquid-container.description = Speichert eine kleine Menge an Flüssigkeiten. Hat Ausgänge an allen Seiten, ähnlich wie ein Flüssigkeitsverteiler.
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1513]: Flüssigkeit, singular. Es kann nur eine Flüssigkeit gespeichert werden (liquid != liquids)

Comment (Catchears) [reply core/assets/bundles/bundle_de.properties:1513]: Müsste man dann auch beim Flüssigkeitstank ändern, aber ich stimme dir zu.

Comment (kuhnchris) [reply core/assets/bundles/bundle_de.properties:1513]: Bin ich auch der Meinung - hab mich am Tank orientiert und habs dort auch schon bescheiden gefunden.

**Comment by kuhnchris** [comment]
@Catchears / @Shawak  *technisch* müsste man es eventuell auch bei den anderen Liquid Blöcken überlegen ob das Wording so passt - kommt eigentlich überall "Flüssigkeiten" vor, auch wenn im Kanal nur eine Flüssigkeit transportiert werden kann... Open to suggestions.

**Pull request merged**